README.md

```diff
@@ -26,7 +26,39 @@ curl https://binaries.hyperliquid.xyz/Testnet/hl-visor > ~/hl-visor && chmod a+x
 ```
 
 ## Running
-Run `~/hl-visor`. It may take a while as your node navigates the network to find an appropriate peer to stream from. Once you see logs like `applied block X` then your node should be streaming live data. You can inspect the transactions or other data as described below.
```

Comment:
Let's keep this and add the screen instructions as a separate section because it isn't strictly necessary